mappings/net/minecraft/world/Difficulty.mapping

```diff
@@ -7,6 +7,7 @@ CLASS net/minecraft/class_1267 net/minecraft/world/Difficulty
 		ARG 4 name
 	METHOD method_16691 byName (Ljava/lang/String;)Lnet/minecraft/class_1267;
 		ARG 0 name
+	METHOD method_27297 getNextDifficulty ()Lnet/minecraft/class_1267;
```

Comment:
Is this `cycle` or `higher` (next higher)? or just `next` (to me this looks like getting the higher difficulty)?

Reply:
It seems to cycle as follows:

```
int newOrdinal = (oldOrdinal + 1) % Difficulty.values().length
```

So cycle probably make sense there

Reply:
```suggestion
	METHOD method_27297 cycle ()Lnet/minecraft/class_1267;
```